### PR TITLE
Docs  footnote page

### PR DESCRIPTION
**Pull request by RichKummer**
### Related Ticket(s)

[#7705](https://app.zenhub.com/workspaces/carbon-for-ibmcom-5d449f3642eb1962336cbe52/issues/carbon-design-system/carbon-for-ibm-dotcom/7705)

### Description

Added a new page for Footnote section containing an overview and anatomy section.

### Changelog

**New**

- New Footnote section page

**Changed**

- Updated yaml file with new page in navigation

**Removed**

- N/A

### CODE REVIEW

File: src/data/components.json [hunk 1]
```diff
@@ -1301,6 +1301,33 @@
...
+    "Footnote section": {
+      "url": "/components/footnote-section",
+      "designLink": "https://ibm.ent.box.com/folder/151933034619",
+      "description": "The Footnote section appears above the Footer and contains a full list of the footnotes used throughout the page.",
+      "storybook": {
+        "webcomponents": "http://www.ibm.com/standards/carbon/web-components/?path=/story/components-footer--default",
+        "react": "http://www.ibm.com/standards/carbon/react/?path=/story/components-footer--default"
+      },
+      "webcomponents": {
+        "stable": true,
```
Comment (oliviaflory) [inline src/data/components.json:1313]:
```suggestion
        "stable": false,
```

File: src/data/components.json [hunk 2]
```diff
@@ -1301,6 +1301,33 @@
...
+        "experimental": false,
+        "new": false,
+        "updated": true,
```
Comment (oliviaflory) [inline src/data/components.json:1316]:
```suggestion
        "updated": false,
```

File: src/data/components.json [hunk 3]
```diff
@@ -1301,6 +1301,33 @@
...
+        "deprecated": false,
+        "underConstruction": false,
```
Comment (oliviaflory) [inline src/data/components.json:1318]:
```suggestion
        "underConstruction": true,
```

File: src/data/components.json [hunk 4]
```diff
@@ -1301,6 +1301,33 @@
...
+        "notAvailable": false
+      },
+      "react": {
+        "stable": true,
```
Comment (oliviaflory) [inline src/data/components.json:1322]:
```suggestion
        "stable": false,
```

File: src/data/components.json [hunk 5]
```diff
@@ -1301,6 +1301,33 @@
...
+        "experimental": false,
+        "new": false,
+        "updated": true,
```
Comment (oliviaflory) [inline src/data/components.json:1325]:
```suggestion
        "updated": false,
```

File: src/data/components.json [hunk 6]
```diff
@@ -1301,6 +1301,33 @@
...
+        "deprecated": false,
+        "underConstruction": false,
```
Comment (oliviaflory) [inline src/data/components.json:1327]:
```suggestion
        "underConstruction": true,
```
I'm guessing they will do a react wrapper for this

File: src/data/components.json [hunk 7]
```diff
@@ -1301,6 +1301,33 @@
         "notAvailable": true
       }
     },
+    "Footnote section": {
+      "url": "/components/footnote-section",
+      "designLink": "https://ibm.ent.box.com/folder/151933034619",
+      "description": "The Footnote section appears above the Footer and contains a full list of the footnotes used throughout the page.",
+      "storybook": {
+        "webcomponents": "http://www.ibm.com/standards/carbon/web-components/?path=/story/components-footer--default",
```
Comment (oliviaflory) [inline src/data/components.json:1309]:
```suggestion
        "webcomponents": "",
```

File: src/data/components.json [hunk 8]
```diff
@@ -1301,6 +1301,33 @@
...
+        "react": "http://www.ibm.com/standards/carbon/react/?path=/story/components-footer--default"
```
Comment (oliviaflory) [inline src/data/components.json:1310]:
```suggestion
        "react": ""
```

**Review by oliviaflory — CHANGES_REQUESTED** [review]
Made some inline comments on the .json file

**Comment by oliviaflory** [comment]
A few things I noticed that we could chat about tomorrow in Huddle:
1. `Footnote section` is throwing me off a bit because all of our `section` components have a Content section title, but we eliminated that bit.. 
2. Overview image: I think it might be helpful to show the overview in the context of a browser using the image production guidelines and the footer below it.
3. Anatomy: Let's add in the optional border to the anatomy spec / list

It might be good to talk about that the list and adjunct information can accommodate links. Can the author/user select just part of the footnote for the link or is the whole thing clickable?

We might want to reference the adjunct links in the footer as well and when to choose the Footnotes vs Footer to house those.

**Comment by RichKummer** [comment]
Made an update to images on the page, but only updated for 2 and 3. 1 and 4 pending

**Pull request closed without merging**